gratipay/models/team.py

```diff
@@ -67,6 +67,26 @@ def insert(cls, owner, **fields):
         """, fields)
 
 
+    def update(self, **kw):
+      updateable = frozenset(['name', 'product_or_service', 'homepage',
+                              'onboarding_url', 'todo_url'])
+
+      cols, vals = zip(*kw.items())
+      assert set(cols).issubset(updateable)
```

Comment:
This would return a `500` if someone tries to force another column in there. Let's throw a `400` instead?